src/kongComponents/model-example.jsx

```diff
@@ -23,8 +23,8 @@ export default class ModelExample extends React.Component {
     }
   }
 
-  activeTab = (e) => {
-    let { target: { dataset: { name } } } = e
+  activeTab =( e ) => {
+    let { target : { dataset : { name } } } = e
```

Comment:
This is some funky styling on the colons, is this a common style in this repo?

Reply:
This is default code from swagger

Reply:
horrifying